src/twisted/internet/interfaces.py

```diff
@@ -1142,82 +1142,83 @@ def spawnProcess(processProtocol, executable, args=(), env={}, path=None,
         Spawn a process, with a process protocol.
 
         @type processProtocol: L{IProcessProtocol} provider
-        @param processProtocol: An object which will be notified of all
-            events related to the created process.
+        @param processProtocol: An object which will be notified of all events
+            related to the created process.
 
         @param executable: the file name to spawn - the full path should be
-                           used.
+            used.
+        @type executable: L{bytes} or L{unicode}
 
         @param args: the command line arguments to pass to the process; a
-                     sequence of strings. The first string should be the
-                     executable's name.
-
-        @type env: a C{dict} mapping C{str} to C{str}, or L{None}.
-        @param env: the environment variables to pass to the child process. The
-                    resulting behavior varies between platforms. If
-                      - C{env} is not set:
-                        - On POSIX: pass an empty environment.
-                        - On Windows: pass C{os.environ}.
-                      - C{env} is L{None}:
-                        - On POSIX: pass C{os.environ}.
-                        - On Windows: pass C{os.environ}.
-                      - C{env} is a C{dict}:
-                        - On POSIX: pass the key/value pairs in C{env} as the
-                          complete environment.
-                        - On Windows: update C{os.environ} with the key/value
-                          pairs in the C{dict} before passing it. As a
-                          consequence of U{bug #1640
-                          <http://twistedmatrix.com/trac/ticket/1640>}, passing
-                          keys with empty values in an effort to unset
-                          environment variables I{won't} unset them.
+            sequence of strings.  The first string should be the executable's
+            name.
+        @type args: L{list} with L{bytes} or L{unicode} items.
+
+        @type env: a L{dict} mapping L{bytes}/L{unicode} keys to
+            L{bytes}/L{unicode} items, or L{None}.
+        @param env: the environment variables to pass to the child process.
+            The resulting behavior varies between platforms.  If::
```

Comment:
hmm yeah. if i get rid of the :: after running python-docstring-mode, it'll be fine?